app/controllers/staff/proposals_controller.rb

```diff
@@ -1,6 +1,8 @@
 class Staff::ProposalsController < Staff::ApplicationController
   before_action :enable_staff_program_subnav
   before_action :set_proposal_counts, only: [:index, :show, :selection]
+  before_action :skip_policy_scope
```

Comment:
I take it this is left over from something you were tinkering with? I don't see any policy scopes being used.